tests/virt_autotest/kubevirt_tests_server.pm

```diff
@@ -241,22 +241,29 @@ sub install_kubevirt_packages {
         transactional::enter_trup_shell(global_options => '--drop-if-no-change') if (is_transactional);
 
         zypper_call("lr -d");
-        zypper_call("ar $virt_tests_repo Virt-Tests-Repo");
         zypper_call("ar $virt_manifests_repo Virt-Manifests-Repo") if ($virt_manifests_repo);
+        zypper_call("ar $virt_tests_repo Virt-Tests-Repo") if ($virt_tests_repo);
         zypper_call("--gpg-auto-import-keys ref");
 
-        $search_manifests = $virt_manifests_pkgs =~ s/\s+/\\\|/gr;
-
         if ($virt_manifests_repo) {
             zypper_call("in -f -r Virt-Manifests-Repo $virt_manifests_pkgs");
-        } elsif (script_run("rpmquery $virt_manifests_pkgs")) {
+        }
+        elsif (script_run("rpmquery $virt_manifests_pkgs")) {
+            $search_manifests = $virt_manifests_pkgs =~ s/\s+/\\\|/gr;
             if (is_transactional || script_run("zypper se -r SLE-Module-Containers${os_version}-Updates $virt_manifests_pkgs | grep -w '$search_manifests'")) {
                 zypper_call("in -f $virt_manifests_pkgs");
             } else {
                 zypper_call("in -f -r SLE-Module-Containers${os_version}-Updates $virt_manifests_pkgs");
             }
         }
-        zypper_call("in -f -r Virt-Tests-Repo $virt_tests_pkg");
+
```

Comment:
How about handling manifest pkg as simple as below virt-test pkg? Above logic may be problematic for sle16, which does not have modules any more.

Reply:
Updated and a latest VR: https://openqa.suse.de/tests/19703976